source/blood/src/gamemenu.cpp

```diff
@@ -182,25 +194,64 @@ void CGameMenuMgr::Draw(void)
 
         m_mousewake_watchpoint = 0;
     }
+
+    vec2_t cursorpos = { (m_mousepos.x >> 16) + 9, (m_mousepos.y >> 16) + 18};
```

Comment:
Shifting coordinate by 16 makes cursor position locked at 320x200 resolution coordinates

Reply:
I'm just shifting it back, a few lines above I was shifting it the other direction.